File: ConnectRealEstateAgent/List/ItemListTableViewCell.m
//
//  ItemListTableViewCell.m
//  ConnectRealEstateAgent
//
//  Created by TAnine on 2023/03/08.
//

#import "ItemListTableViewCell.h"

@implementation ItemListTableViewCell

- (void)awakeFromNib {
    [super awakeFromNib];
    // Initialization code
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

- (void)initCellLbl:(NSString *)addr sell:(NSString *)sell income:(NSString *)income year:(NSString *)year rat:(NSString *)rat {
    [self.addrLbl setText:addr];
    [self.sellLbl setText:sell];
    [self.incomeLbl setText:income];
    [self.yearIncomeLbl setText:year];
    [self.ratLbl setText:rat];
    
}
@end

File: ConnectRealEstateAgent/List/ItemListTableViewCell.swift
//
//  ItemListTableViewCell.swift
//  ConnectRealEstateAgent
//
//  Created by TAnine on 2023/03/08.
//

import UIKit

class ItemListTableViewCell: UITableViewCell {
    
    @IBOutlet weak var addressLbl: UILabel!
    @IBOutlet weak var sellLbl: UILabel!
    @IBOutlet weak var incomeLbl: UILabel!
    @IBOutlet weak var ratLbl: UILabel!
    

    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }

    override func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }
    
//    func setupLabel(addr: String, sell: String, income: String, rat: String){
//        self.addressLbl.text = addr
//        self.sellLbl.text = sell
//        self.incomeLbl.text = income
//        self.ratLbl.text = rat
//    }
}
